lua_modules/ftp/server.lua

```diff
@@ -0,0 +1,157 @@
+-- a simple ftp server
+USER = "test"
+PASS = "12345"
+local file,net,pairs,print,string,table = file,net,pairs,print,string,table
+data_fnc = nil
```

Comment:
```Lua
local data_fnc = function() end
```
and then the listener becomes `ftp_data:listen(20, function (s) data_fnc(s) end)`  You still need the extra closure because you overwrite data_fnc/

Reply:
and owerwrite with blank function after STOR,RETR is comlete?